src/app/app.component.html

```diff
@@ -1,4 +1,5 @@
 <app-pizza-toppings></app-pizza-toppings>
+<app-asz-random-toppings></app-asz-random-toppings>
```

Comment:
Can you move your component to the bottom of the page here? Then I think it would be ready for me to accept/merge. Thanks.